app/templates/Event/_common/event_attendees.html.twig

```diff
@@ -0,0 +1,17 @@
+{% if (event.getAttendeeCount  > 0) %}
+<h3>Who's attending?</h3>
+    <div class="row">
+        {% for attendee in attendees %}
+            <div class="col col-sm-4 col-xs-3">
```

Comment:
Wouldn't it be possible to set this to ```col-lg-3 col-md-4 col-sm-6 col-xs-12```? Then it should be somewhat responsive and we might solve the question @akrabat raised regarding scalability for 100 users :wink: 

Alternative would be a list with the ```li``` styled as inline-block.

I'D also here prefer a styling using ```<ul>``` as it's a list here, but that's absolutely minor.